lib_forceload/data/gm4_forceload-1.0/functions/update_marker.mcfunction

```diff
@@ -0,0 +1,9 @@
+# @s = area effect cloud dimension marker
```

Comment:
The convention from other modules is to call this function `upgrade_marker`

Reply:
Fixed it in the latest commit